File: src/main/entity/Person.java
package entity;

import java.io.Serializable;

public class Person implements Client, Serializable {

    /** Stores all values as subclasses
     * Getters return all values as Strings
     */

    private final Name name;
    private final Phone phone;
    private final Email email;


    public Person(Name name, Phone phone, Email email) {
        this.name = name;
        this.phone = phone;
        this.email = email;
    }

    public String getName() {
        return name.getFullName();
    }

    public String getPronouns() {
        return name.getPronouns();
    }

    public String getPhone() {
        return phone.getPhone();
    }

    public String getEmail() {
        return email.getEmail();
    }

}

File: kard_project_test/lib/person.dart
class Person {

  String name;
  String phone;
  String email;
  String pronouns;

  Person(
      this.name,
      this.phone,
      this.email,
      this.pronouns);

  Person.fromJson(Map<String, dynamic> json):
      name = json['name'],
      phone = json['phone'],
      email = json['email'],
      pronouns = json['pronouns'];
}